wolfssl/wolfcrypt/settings.h

```diff
@@ -4403,6 +4403,10 @@ extern void uITRON4_free(void *p) ;
     #endif
 #endif /* HAVE_ENTROPY_MEMUSE */
 
+#ifndef WANT_BASE64_TABLE
```

Comment:
Do we even need to support the old school table method anymore? @SparkiDev can we just rip out the old `base64Decode` table and always use the constant time `Base64_Char2Val` now?

Reply:
Leave the old one there.
The attack is not simple and the table is faster.

Reply:
What @SparkiDev said.  We are eventually going to build both simultaneously, and use the fast one for decoding non-sensitive data (e.g. PEM certificates) where the speed can be very helpful.